index.rst

```diff
@@ -86,10 +93,15 @@ Initial user authentication
 Initial user authentication for most deployments will be done via CILogon using a web browser.
 CILogon will be used as an OpenID Connect provider, so the output from that authentication process will be a JWT issued by CILogon and containing the user's identity information.
 
-The CILogon-provided identity will be mapped to a Science Platform user.
-Users will be able to associate multiple CILogon identities with the same Science Platform user.
+If the user has not previously been seen, they may or may not be able to automatically create a Science Platform account, depending on their institutional affiliation.
```

Comment:
I think it would be appropriate to footnote this discussion with a reference to LSE-279.